src/VolumeLogger.js

```diff
@@ -38,12 +39,13 @@ module.exports = class VolumeLogger {
         const kbOutPerSecond = (this.outBytes / this.reportingIntervalSeconds) / 1000
 
         const networkMetrics = await this.networkNode.getMetrics()
+        const connectionCount = this.connectionCountWS + this.connectionCountMQTT
 
         this.lastVolumeStatistics = {
             id: this.networkNode.opts.id,
             timestamp: Date.now(),
             totalBufferSize: this.totalBufferSize,
-            numOfOpenWebsockets: this.connectionCount,
+            connectionCount,
```

Comment:
Would it make sense to also have separate statistics for `mqtt` and `ws` counts? That way we can follow statistics of how users are using our different adapters.